docs/source/get_started/install.md

```diff
@@ -1,102 +1,192 @@
-# Install Kedro
+# Set up Kedro
 
-If you encounter any problems as you install Kedro, you can ask for help on Kedro's [Slack organisation](https://slack.kedro.org) or [search the archives for a solution](https://linen-discord.kedro.org).
+## Summary
+
+* Kedro can be used on Windows, macOS or Linux.
+* Installation prerequisites include Python 3.7+, a virtual environment manager like `conda`, and `git`.
+* You should install Kedro using `pip install kedro`.
+
+If you encounter any problems as you set up Kedro, you can ask for help on Kedro's [Slack organisation](https://slack.kedro.org) or [search the archives for a solution](https://linen-discord.kedro.org).
 
 
 ## Installation prerequisites
+* **Python**: Kedro supports macOS, Linux, and Windows and is built for Python 3.7+. You'll select a version of Python when you create a virtual environment for your Kedro project.
+
+* **Python virtual environment**: We suggest you create a new Python virtual environment for *each* new Kedro project you work on to isolate its dependencies from those of other projects.
+
+* **git**: You must install `git` onto your machine if you do not already have it. Type `git -v` into your terminal window to confirm it is installed. You should see the version of `git` available or an error message. You can download it from the official  [`git`](https://git-scm.com/) website.
+
+## Create a Python virtual environment for your Kedro project
+
+We strongly recommend [installing `conda` as your virtual environment manager](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) if you don't already use it.
+
+``` {tip}
+[Read more about Python virtual environments](https://realpython.com/python-virtual-environments-a-primer/) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
+```
+
+### How to create a new virtual environment using `conda`
+
+This is our recommended approach. From your terminal:
+
+```bash
+conda create --name kedro-environment python=3.10 -y
+```
+
+In this example, we use Python 3.10, and create a virtual environment called `kedro-environment`. You can opt for a different version of Python (any version >= 3.7 and <3.11) for your project, and name it anything you choose.
```

Comment:
I get that _we_ are setting some recommendations here (rather than enforcing readers to do a specific thing) but I'm having a hard time reading all the first-person sentences https://ddbeck.com/first-person-plural-is-questionable/

Wondering if we could rephrase to reduce those, what do you think?

Reply:
Agreed! I'll do a final grammatical run through.